packages/graphql-typescript-definitions/src/index.ts

```diff
@@ -158,6 +160,17 @@ export class Builder extends EventEmitter {
     this.emit('start');
     let ast: AST;
 
+    const duplicateNames = getDuplicateOperationNames(this.documentCache);
+
+    if (duplicateNames.length) {
+      duplicateNames.forEach((name) => {
+        const error = new Error(
+          `Graphql operations must have unique names. Found 2 or more operations with the name "${name}"`,
```

Comment:
missing a period

Reply:
If we have file info available, we shouldn't make users hunt down specific files.  An error like this provides good context for self-service:

<img width="635" alt="neutron zsh 2018-08-26 14-28-03" src="https://user-images.githubusercontent.com/673655/44631590-512f8200-a93c-11e8-8d90-fd389e833dce.png">


Scrappy implementation is inside the details block:
<details>
```ts
  const duplicateOperations = getDuplicateOperations(this.documentCache);
  if (duplicateOperations.length) {
    duplicateOperations.forEach(({name, files}) => {
      const error = new Error(
        `Graphql operations must have a unique name. The operation "${name}" is declared in:\n    ${files.join(
          '\n    ',
        )}`,
      );
      this.emit('error', error);
    });
  }

function getDuplicateOperations(documents: Map<string, DocumentNode>) {
  const operations = new Map<string, Set<string>>();

  Array.from(documents.entries()).forEach(([path, document]) => {
    document.definitions.filter(isOperation).forEach((definition) => {
      const {name} = definition as OperationDefinitionNode;
      if (name && name.value) {
        const map = operations.get(name.value);
        if (map) {
          map.add(path);
        } else {
          operations.set(name.value, new Set([path]));
        }
      }
    });
  });

  return Array.from(operations.entries())
    .filter(([, files]) => files.size > 1)
    .map(([operation, files]) => {
      return {name: operation, files: Array.from(files)};
    });
}
```
</details>

Reply:
In watch mode, we shouldn't generate `.d.ts` files for clashing files.